packages/inter-protocol/test/psm/test-governedPsm.js

```diff
@@ -44,14 +44,129 @@ test('psm block offers w/Governance', async t => {
 
   t.deepEqual(['wantMinted'], await E(zoe).getOfferFilter(psm.instance));
 
-  const giveCentral = AmountMath.make(knutIssuer.brand, 1_000_000n);
+  const giveCentral = AmountMath.make(knutIssuerKit.brand, 1_000_000n);
```

Comment:
e.g. if this were amount utils,
```suggestion
  const giveCentral = knut.make(1_000_000n);
```

Reply:
done